core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3368,9 +3368,9 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
-  public static final PropertyKey USER_METADATA_CACHE_ENABLED =
-      new Builder(Name.USER_METADATA_CACHE_ENABLED)
-          .setDefaultValue(false)
+  public static final PropertyKey USER_FILESYSTEM_CLASS =
+      new Builder(Name.USER_FILESYSTEM_CLASS)
+          .setDefaultValue("alluxio.client.file.BaseFileSystem")
```

Comment:
Do you think we should list out the different options this can be configured with?

Reply:
Good catch, I forgot the change the description.